ql/src/test/org/apache/hadoop/hive/ql/io/parquet/TestHiveSchemaConverter.java

```diff
@@ -431,7 +431,8 @@ public void testMapOriginalType() throws Exception {
     Type secondLevel = topLevel.asGroupType().getFields().get(0);
     // there is one repeated field for mapCol, the field name is "map" and its original Type is
     // MAP_KEY_VALUE;
-    checkField(secondLevel, "map", Repetition.REPEATED, LogicalTypeAnnotation.MapKeyValueTypeAnnotation.getInstance());
```

Comment:
BTW, do you know which Parquet JIRA made this change?

Reply:
I don't know yet. Checked all the JIRAs in 1.11.1 and it's not obvious which one caused it. I'll post my finding later.

Reply:
Thanks!